mule-maven-plugin/README.adoc

```diff
@@ -36,547 +34,178 @@ Add https://repository.mulesoft.org/nexus/content/repositories/releases[this] re
     </configuration>
-    <executions>
-        <execution>
-            <id>deploy</id>
-            <phase>deploy</phase>
-            <goals>
-                <goal>deploy</goal>
-            </goals>
-        </execution>
-    </executions>
 </plugin>
 ----
 
-The _uri_ parameter is the endpoint of the REST API of the Agent.
-
-
-== Running Integration Tests
-
-One of the most important uses for the plugin is to run integration tests on your integration application. You can see the working example in src/it/standalone/example-integration-tests.
-
-Basically you will configure _maven-mule-plugin_ to pack your project in Mule app format, _maven-failsafe-plugin_ to run integration-tests and report, and this plugin to deploy the project packaged application to a new Mule Server downloaded from a Maven repository. 
-
-[source,xml]
-----
-<plugins>
-    <plugin>
-        <groupId>org.mule.tools.maven</groupId>
-        <artifactId>mule-app-maven-plugin</artifactId>
-        <version>1.1</version>
-        <extensions>true</extensions>
-    </plugin>
-    <plugin>
-        <groupId>org.mule.tools.maven</groupId>
-        <artifactId>mule-maven-plugin</artifactId>
-        <version>2.0</version>
-        <configuration>
-            <deploymentType>standalone</deploymentType>
-            <muleVersion>3.7.0</muleVersion>
-        </configuration>
-        <executions>
-            <execution>
-                <id>deploy</id>
-                <phase>pre-integration-test</phase>
-                <goals>
-                    <goal>deploy</goal>
-                </goals>
-            </execution>
-            <execution>
-                <id>undeploy</id>
-                <phase>post-integration-test</phase>
-                <goals>
-                    <goal>undeploy</goal>
-                </goals>
-            </execution>
-        </executions>
-    </plugin>
-    <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-failsafe-plugin</artifactId>
-        <executions>
-            <execution>
-                <id>integration-test</id>
-                <goals>
-                    <goal>integration-test</goal>
-                    <goal>verify</goal>
-                </goals>
-            </execution>
-        </executions>
-    </plugin>
-</plugins>
-----
+=== Notes
 
-In this example, the _mule-app-maven-plugin_ packages the Mule application that is deployed by the _mule-maven-plugin_.
+* You need to have a Mule Standalone Runtime Server installed locally and pointt to it with the *muleHome* property.
 
 
-== Full example
+== Deploy to Anypoint Runtime Manager
 
-In this example you will see the plugin working for Standalone deployment, configuring one applications to be deployed, two external libs to be added to the server, a domain to be deployed, and a script to be ran just before starting the server.
+You can deploy your application to a running ARM server, serverGroup or cluster.
```

Comment:
should serverGroup go separately (server group)?

Reply:
done